microsoft-edge/toc.yml

```
@@ -1259,10 +1241,53 @@
               href: webview2/how-to/print.md
               displayName: Printing from WebView2 apps # full title
 
-            # updated h2: Experimental APIs, added a 2nd, new h3:
-            - name: Understand the different WebView2 SDK versions
+            - name: Working with local content in WebView2 apps
+              href: webview2/concepts/working-with-local-content.md
+              displayName: 
+
+            # - name: Customize the UI   # not needed?  flatten
+            #   items:
+            #     - name: Customize the UI of WebView2 apps
+            #       href: webview2/how-to/customize-ui.md
+            #       displayName: context menu, right-click menu
+
+            - name: Customize context menus
+              href: webview2/how-to/context-menus.md
+              displayName: right-click menu, Customize context menus in WebView2  # top-of-page title
+
+# -----------------------------------------------------------------------------
+        - name: Web and native interop
+          items:
+            - name: Interop of native-side and web-side code
```

```suggestion
            - name: Interop of native and web code
```
@aluhrs13 - Are the terms "native-side code" and "web-side code" used in the industry?  Affects:

TOC article titles near "AddHostObject"
Wording in those articles.
Wording on right side of large diagram.  We had -side, then removed it:

Rendered article section showing the large diagram which used to have "-side" in about 4 boxes on the right:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/index?branch=pr-en-us-2450#overview-of-the-components-of-the-webview2-platform  [⁠[gh rendered](https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/wv2-intro-diagram/microsoft-edge/webview2/index.md#overview-of-the-components-of-the-webview2-platform)]

Web searches:
https://www.bing.com/search?pglt=129&q=%22native-side+code%22
https://www.bing.com/search?pglt=129&q=%22web-side+code%22

---

retitled article in https://github.com/MicrosoftDocs/edge-developer/pull/2725